tools/cvm-image-rewriter/plugins/03-netplan/README.md

```diff
@@ -0,0 +1,3 @@
+# Netplan
+
+This plugin is used to add network configuration file netplan.yaml in the `/etc/netplan/` directory, which will configure the network for the guest image.
```

Comment:
If the user wants to customize netplan.yaml, it needs to update the content of "/etc/netplan/netplan.yaml"